_maps/giantbeanstalk/giantbeanstalk.yaml

```diff
@@ -0,0 +1,186 @@
+---
+name:
+  en: Giant Beanstalk
+desc:
+  en: Used to reach places Mario can't usually go, this is not your ordinary beanstalk. Take a climb to the top and enjoy the view.
+ruleSet: Standard
+theme: Mario
+initialCash: 2200
+targetAmount: 20000
+baseSalary: 275
+salaryIncrement: 250
+maxDiceRoll: 6
+frbFile1: giantbeanstalk
+looping:
+  mode: Vertical
+  radius: 28
+  horizontalPadding: .95
+  verticalSquareCount: 17
+background: bg008alt
+mapIcon: gbs
+music:
+  download:
+  - https://nikkums.io/cswt/BGM/GiantBeanstalk.music.zip
+  - https://drive.google.com/file/d/1PujpGrdxuR0dF_x8xGaqJ2se3__mYjJ0/view?usp=drive_link

```

Comment:
This Google Drive download link should be: `https://drive.google.com/uc?export=download&confirm=1&id=1PujpGrdxuR0dF_x8xGaqJ2se3__mYjJ0`